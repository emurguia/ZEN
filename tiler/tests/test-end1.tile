// tests that end block works as expected

int count;
init{
	count = 0;
	tile(3,3);
}


turn{
	iprint(count);
	count = count + 1;
}


end{
	if (count > 2) {
		close();
		return 1;
	}
}