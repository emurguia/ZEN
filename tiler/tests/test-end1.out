0
1
2
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.